infra/resources/_modules/cdn/dns.tf

```
@@ -17,4 +17,28 @@ resource "azurerm_dns_txt_record" "wallet_io_pagopa_it" {
     value = "qidvirtenu"
   }
   tags = var.tags
+}
+
+# TXT for cbor Maven namespace verification
+resource "azurerm_dns_txt_record" "cbor_wallet_io_pagopa_it" {
+  name                = "@"
+  zone_name           = local.wallet_cdn_dns_zone_name
+  resource_group_name = local.dns_zone_name_resource_group_name
+  ttl                 = 3600
+  record {
+    value = "mku7fqer6d"
+  }
+  tags = var.tags
+}
+
+# TXT for proximity Maven namespace verification
+resource "azurerm_dns_txt_record" "proximity_wallet_io_pagopa_it" {
+  name                = "@"
```

I think the name of the record should coincide with the namespace (proximity.wallet.io.pagopa.it)

```suggestion
  name                = "proximity"
```